docs/userGuide/usingHtmlJavaScriptCss.md

```diff
@@ -61,5 +63,81 @@ Alternatively, you can use `<markdown>` (for _block_ Markdown elements such as h
 
 </div>
 
+## JavaScript chart libraries
```

Comment:
Does this need to be narrowed to chart libraries or write it in general for any JS library but use chart libraries as an example only?